community/kernel/src/main/java/org/neo4j/kernel/SchemaRuleVerifier.java

```diff
@@ -17,26 +17,13 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.kernel.impl.coreapi.schema;
+package org.neo4j.kernel;
```

Comment:
This is most likely not the right package for this interface.

We do not want to include this interface in the public API, and types in this package become part of the public API.

How about making it a neighbour of `SchemaRule`, the thing that it verifies?